### DISCUSSION TITLE
Integrate Airtable with Elementor

### DISCUSSION THREAD

**Discussion by irixzafra** [elementor/elementor#29157] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

Managing dynamic data across platforms is challenging for Elementor users needing frequent updates, especially without native Airtable integration. Airtable’s user-friendly database is ideal for managing product inventories, events, and client information, but currently, updating Elementor websites with Airtable data requires third-party plugins or custom API setups that are often complex and unreliable. A native Airtable integration with Elementor dynamic fields would streamline this process, making it easy to maintain data-driven content directly within Elementor without additional tools.

### Describe the solution you'd like.

The solution would be a direct integration between Elementor’s dynamic fields and Airtable. This feature would allow Elementor users to pull data directly from Airtable fields and databases, populating Elementor widgets with updated Airtable content in real-time. For example, users could display dynamic lists, galleries, and tables directly sourced from Airtable without additional coding or plugins.

### Describe alternatives you've considered.

Alternative solutions involve using third-party plugins or custom API setups to connect Airtable with Elementor. However, these options may require advanced technical skills, are more prone to errors, and increase dependency on external software that may not integrate smoothly.

### Additional context

An example use case includes an e-commerce business that manages its product inventory in Airtable and wants to display available products on their Elementor-built website. With this integration, any updates in Airtable (like stock changes or new items) would instantly appear on the website without manual updates or additional plugins.

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by toineenzo** [comment]
Have you tried using the Airtable Webhook API? You maybe could use that with Elementor Forms
